File: ToyInterpreter 2/src/Model/IDictionary.java
package Model;

import mobserver.VarTableObserver;

public interface IDictionary {
	public int get(String n);
	public void set(String n,int v);
	public void printTable();
	public String toString();
	public void addObserver(VarTableObserver o);
	public IDictionary clone();
}

File: ToyInterpreter/ToyInterpreter/Model/IDictionary.cs
using System;
using ToyInterpreter.Model.MObserver;

namespace ToyInterpreter.Model
{
	public interface IDictionary {
		int get(String n);
		void set(String n,int v);
		void printTable();
		void addObserver (VarTableObserver o);
		String toString();
		IDictionary clone();
	}
}

